#Previous Seminars

##Semester 25|26

<div class="accordion">
  <button class="accordion-button">Jan 14, 2026 - Edda Schulz, MPI für Molekulare Genetik, Berlin</button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong>From gene titration to dosage compensation: principles of quantitative gene control</strong><p>

    <p> Precise quantitative control of gene activity is essential for human physiology, as evidenced by numerous haplo-insufficient and triplo-sensitive genes and the evolution of sex chromosome dosage compensation mechanisms like X-chromosome inactivation. 
    To experimentally dissect dosage sensitivity, we developed CasTuner, a new epigenome engineering tool that can titrate endogenous gene expression to reveal how dose information is decoded by DNA elements. 
    Here we use CasTuner to investigate how X-chromosomal dosage is sensed to restrict X inactivation to females. Through high-throughput screens, we identified dosage-sensitive X inactivation regulators that govern this female-specificity. To understand how multiple regulators cooperate, we employ dynamic mathematical modeling to compare several model structures for their ability to ensure female-specificity using stochastic simulations. 
    By comparing model simulations with experimental measurements, we reveal a new concept: Two X-linked genes acting as a dosage sensor and feedback regulator cooperate to ensure robust female-specific X inactivation.</p>
  </div>
</div>

<div class="accordion">
  <button class="accordion-button">Dec 3, 2025 - Mohit Kumar Jolly, Ph.D, Indian Institute of Science, Bangalore</button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong>Interconnected axes of phenotypic plasticity drive cancer cell fitness and worse clinical outcomes</strong><p>

    <p>Phenotypic plasticity is a key driver of metastasis and drug resistance, as it enables cancer cells to adapt reversibly to varying environments without altering their genomic architecture. 
    Different axes of phenotypic plasticity – for instance, metabolic plasticity, lineage switching, immune evasion
    – have been investigated stand-alone, but the mechanisms and implications of their coordination remains unclear. 
    Here, integrating mechanism-based mathematical modeling of underlying regulatory networks with bulk and single-cell transcriptomic data analysis in ER+ breast cancer, we show that these various axes of plasticity function in a coordinated manner, such that the network is composed of two mutually antagonistic “teams” of nodes supporting opposite cellular behaviors. One ‘team’ supports glycolytic, stem-like, hybrid epithelial/mesenchymal or mesenchymal, drug-resistant, and basal-like cell-state, while the other ‘team’ supports oxidative phosphorylation, non-stem-like, epithelial, drug-sensitive, and lumina-like cell-state. Importantly, coordinated activation of more than one axis of plasticity, but not for each axis alone, associates with worse clinical outcomes in TCGA and METABRIC ER+ breast cancer cohorts. Together, these results suggest that cancer cell fitness is enhanced by coordinated switching of cell-state along multiple axes of plasticity simultaneously.</p> 

    <p><strong>Biosketch</strong></p>

    <p>Mohit is an Associate Professor in Department of Bioengineering at Indian Institute of Science, Bangalore.</p> 
    He obtained his Bachelors and Masters from IIT Kanpur and PhD from Rice University, all in Bioengineering.<br>
    <p>His group integrates multi-scale mathematical models and single-cell multi-omics analysis to understand the dynamics of phenotypic plasticity and heterogeneity driving cancer metastasis and drug resistance. 
    He has been honoured as an Outstanding Young Alumnus of both his alma maters – IIT Kanpur (2022) and Rice University (2024). 
    He has received the prestigious ICTP (International Center for Theoretical Physics) 2023 Prize and the coveted Arthur Winfree Prize 2025 by the Society for Mathematical Biology. He is the current Editor-in-Chief of NPJ Systems Biology & Applications.</p>
  </div>
</div>

<div class="accordion">
  <button class="accordion-button">Nov 5, 2025 - Kathrin Thedieck, Essen University</button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Systems approaches to metabolic signaling in cancer: integrating nutrient networks, drug interventions and environmental chemicals to develop predictive markers for therapy response and quality of life</strong><p>

    <p> Prof. Dr. Kathrin Thedieck, Lab for Metabolic Signaling  

    <p> (1) Department Metabolism, Senescence and Autophagy, Research Center One Health Ruhr, University Alliance Ruhr & University Hospital Essen, University Duisburg-Essen, Germany.<p>

    <p> (2) Center of Medical Biotechnology, Faculty of Biology, University Duisburg-Essen, Germany.<p>
  
    <p> (3) German Cancer Consortium (DKTK), partner site Essen/Duesseldorf, Heidelberg, Germany.<p>

    <p> (4) West German Cancer Center, University Hospital Essen, Essen, Germany.<p>

 

    In the Lab for Metabolic Signaling at the Research Center One Health Ruhr, we investigate how environmental factors – in particular nutrients and chemical stressors – influence cellular signaling, metabolism and stress responses. Focusing on signaling networks centered on the metabolic master regulator and cancer driver mTOR (mechanistic target of rapamycin), our central goal is to understand the mechanisms via which environmental cues modulate disease course and therapy response in cancer and rare neurodevelopmental conditions. Our research is at the center of a modern molecular understanding of One Health, aiming to elucidate the mode of action via which nutrient status, drug therapies and environmental chemicals that act on the hormone system and metabolism (so called metabolism disrupting chemicals, MDC) impinge on disease-relevant signaling networks and harm human health. To tackle this challenge, we take a multidisciplinary approach that combines mechanistic cell biological and biochemical studies with the development of new analytical approaches for integrated proteo-metabolomics, and systems modeling to integrate data at mechanistic and multi-omics scale [1,2]. With the H2020 MESI-STRAT and BMFTR BALANCE-ET consortia we investigate molecular mechanisms of metabolic signaling and therapy response. In longitudinal breast cancer cohorts, we study the linkage between shifted tumor and whole-body metabolism and long term survivorship, relapse, and quality of life during and after endocrine therapies. Within the PARC consortium, we discover mechanisms of action for MDCs and translate our findings to analytical methods for medium and high throughput screening of the metabolism disrupting potential of chemicals [3]. I will present our latest analytical developments and examples for their application.

    <p> Literature:<p>  

    <p> [1] Finding new edges: systems approaches to MTOR signaling. Heberle et al.. Biochem Soc Trans. 2021 doi: 10.1042/BST20190730. PMID: 33544134<p>

    <p> [2] ODE-based models of signaling networks in autophagy. Galhuber M & Thedieck K Current Opinion in Systems Biology, 2024, doi.org/10.1016/j.coisb.2024.100519.<p>

    <p> [3] Development of new approach methods for the identification and characterization of endocrine metabolic disruptors-a PARC project. Braeuning et al. Front Toxicol. 2023. doi:[10.3389/ftox.2023.1212509](10.3389/ftox.2023.1212509). PMID: 37456981<p>

    <p>Visit <a href="https://metabolic-signaling.eu/">Metabolic-Signaling</a> to find out more about our work.</p>

    <p><strong>Biosketch</strong></p>  

    <p>Kathrin Thedieck</p>  

    <p><strong>Lab for Metabolic Signaling</strong></p>  

    <p>Department Metabolism, Senescence and Autophagy, Research Center One Health Ruhr, University Alliance Ruhr & University Hospital Essen, University Duisburg-Essen, Germany Freiburg Materials Research Center (FMF), University of Freiburg, Germany</p>

    <p><strong>correspondence</strong><a href="mailto:kathrin.thedieck@uni-due.de">kathrin.thedieck@uni-due.de</a> </p>

    <p>K. Thedieck and her team pioneer systems approaches to investigate metabolic signaling through the kinase mTOR, a master regulator of metabolic control in cancer. 
    Her research combines biochemical, analytical and computational methodologies to translate mechanistic understanding into marker panels to detect metabolic stress and improve therapy outcomes.</p>

    <p> Thedieck studied in Münster (DE) and Strasbourg (FR) and received her PhD degree in 2005 at the Helmholtz Centre for Infection Research (HZI) in Braunschweig (DE). 
    As a postdoc she worked with Michael N. Hall in Basel (CH) and became in 2008 group leader at the University of Freiburg (DE). 
    In 2013, she joined the University of Groningen (NL) as an Adjunct Professor for metabolic signaling. In 2019, she chose to accept the chair for Biochemistry at Innsbruck University (AT). 
    Since 2024, K. Thedieck heads the Department for Metabolism, Senescence and Autophagy at the Research Center One Health of the Ruhr Alliance and University Clinic Essen (DE). 
    She continues to coordinate systems oncology consortia, among them the H2020 MESI-STRAT and BMFTR BALANCE-ET focusing on tumor metabolism and personalized therapies in breast cancer. 
    K. Thedieck received numerous calls for professorship, fellowships and prizes including awards from the German TS Foundation (2017) and the Klüh foundation (2024) for her research into new therapies for Tuberous Sclerosis Complex, 
    and the ERC Advanced Grant BEYOND STRESS.</p>

    <p><strong>Find out more on</strong> <a href="https://www.metabolic-signaling.eu/">www.metabolic-signaling.eu</a>.</p>
  </div>
</div>

## Semester 24|25
<div class="accordion">
  <button class="accordion-button">June 3, 2025 - Uri Alon - Weizmann Institute, Israel</button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Simplifying inflammation and fibrosis </strong><p>

    <p>Fibrosis – or excess scarring – is pathology that cuts across medicine. Preventing and reversing fibrosis is an unmet medical need. 
    In this whiteboard talk, we will present experiments guided by a mathematical model to understand principles of fibrosis. 
    We will see how this helps to identify new targets, which led to two preclinical treatments that reverse fibrosis in mice heart and liver.</p>

    <p><strong>Biosketch:</strong></p> 
    <p><strong>Education:</strong></p>
    <p>1997-1999 Post-Doctoral Fellow with S. LeiblerDept. Molecular Biology,<br>
    Princeton University1992-1996 Ph.D., Physics, <br>
    Weizmann Institute of Science1989 M.Sc., Physics, <br>
    Hebrew University of Jerusalem1986-1989 B.Sc., Physics and Mathematics, <br>
    Hebrew University of Jerusalem</p>
    <p><strong>Employment:</strong></p>
    <p>2000-2004 Senior Scientist, Dept. Molecular Cell Biology (joint membership at Dept. Physics of Complex Systems), <br>
    Weizmann Institueof Science 2004 -Associate Professor, Dept. Molecular Cell Biology (joint membership at Dept. Physics of Complex Systems), <br>
    Weizmann Institueof Science 2008-2010 Visiting Professor, Dept. Systems Biology, <br>
    Harvard Medical School 2008-Full Professor, Dept. Molecular Cell Biology, <br>
    Weizmann Institute of Science</p>
    
    <p><strong>Teaching experience:</strong></p>
    <p>2000-2008 Developed and taught 'Systems Biology' graduate course, together with N. Barkai, for biology, physics and bioniformaticstracks, Weizmann Institute of Science </p>
    <p><strong>Research Interests:</strong></p>
    <p>Systems Biology. Experimental and theoretical study of protein networks. Design principles of genetic circuits. Physics of biological systems.</p>
  </div>
</div>

<div class="accordion">
  <button class="accordion-button">May 7, 2025 - Barak Raveh - The Hebrew University of Jerusalem, Israel<br> <strong>including video </strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Bayesian metamodeling of complex biomolecular processes in living cells </strong><p>

    <p>Bayesian metamodeling is a divide-and-conquer approach for the integrative modeling of complex biological systems and processes. 
    Instead of constructing a single centralized model based on all available data, we statistically couple a set of input models, each leveraging distinct data types and mathematical representations. 
    Through this modularity, we can combine models that describe different facets of the system across multiple scales and levels of granularity. 
    For example, we can integrate a network model based on protein-protein interaction data, a molecular simulation based on biochemical data, and a mathematical model based on electrophysiological data. 
    We illustrate the benefits and practical challenges of Bayesian metamodeling for modeling two complex cellular processes. 
    In the context of transport between the cytoplasm and the nucleus, Bayesian metamodeling bridges the gap between transport through individual nuclear pore complexes and an entire cell, thereby resolving disparities between experimental data and computational predictions. 
    In the context of T-cell signaling, Bayesian metamodeling unravels the dynamic nanoscale organization of molecules at the immune synapse during the initial stages of T-cell activation by foreign antigens, as discerned from high-resolution imaging data, revealing microscopic mechanisms underlying swift, accurate, precise, and robust immune response. 
    By embracing a decentralized approach to the integrative modeling process, Bayesian metamodeling yields more comprehensive, precise, and accurate descriptions, effectively contextualizing input models, resolving conflicting information, and, crucially, 
    fostering collaborative scientific inquiry by providing a robust framework for sharing expertise, resources, data, and models.</p>

    <p><strong>Biosketch:</strong></p> 
    <p>Barak Raveh is a senior researcher in the School of Computer Science and a member of the Computational Biology Program at the Hebrew University of Jerusalem, where his lab develops integrative, multiscale modelling approaches for dynamic biological systems. 
    Previously, he completed his post-doctoral training with Andrej Sali at the University of California, San Francisco, developing integrative modeling methodologies and applying then to study nucleocytoplasmic transport. 
    He earned his PhD jointly at the Hebrew University’s Department of Molecular Genetics and Tel Aviv University’s School of Computer Science, working with Ora Schueler-Furman and Dan Halperin on Rosetta-based algorithms for peptide–protein docking and protein motion. 
    Earlier, he obtained an MSc in Computational Biology from the Weizmann Institute under Gideon Schreiber and Ronen Basri, following a BSc in Computational Biology from the Hebrew University.</p>
    
    <p><strong>Video embedden!!!</strong></p> 
  </div>
</div>


<div class="accordion">
  <button class="accordion-button">April 2, 2025 - Stefan Legewie - University of Stuttgart </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Modeling the deregulation of alternative splicing in disease </strong><p>

    <p>Alternative pre-mRNA splicing is an essential step in human gene regulation, and mutation- induced aberrant splicing is frequently found in diseases and therapy resistance. 
    Splicing regulation is highly dependent on sequence and cellular context, posing a challenge to predict outcomes of splicing-related disease mutations. 
    In my talk, I will outline how we use kinetic modeling to derive the underlying quantitative principles, describing splice site competition and downstream effects on a wide spectrum of splice isoforms. 
    These models are trained based on high-throughput mutagenesis and genome-wide RNA sequencing data to identify cis-acting mutations and trans-acting RNA-binding proteins that contribute to tumorigenesis, cancer therapy resistance and retinopathies. 
    Taken together, our mechanistic models provide insights into context-dependent complex isoform changes and cryptic splice site activation in health and disease.</p>

    <p><strong>Biosketch:</strong></p> 
    <p>Stefan Legewie studied Biochemistry and did his PhD in Biophysics with Hanspeter Herzel at the Humboldt University of Berlin, focusing on mathematical modeling of signaling pathways. 
    After a brief stay at the German Cancer Research Center, he moved to the newly founded Institute of Molecular Biology (IMB) in Mainz as an independent research group leader. 
    Since 2020 he is Professor for Systems Biology at the University of Stuttgart. His research focuses on the quantitative analysis of gene expression, covering the regulation of alternative splicing and cellular decision making in the TGFb/SMAD signaling pathway.</p>

  </div>
</div>


<div class="accordion">
  <button class="accordion-button">March 12, 2025 - Birgit Schoeberl - Novartis BioMedical Research, Cambridge, USA<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> The Critical Role of Mechanistic Disease Understanding in Drug Development </strong><p>

    <p> In the rapidly evolving landscape of drug development, a deep mechanistic understanding of disease biology is paramount. 
        QSP modeling integrates biological, pharmacological, and clinical data to create comprehensive, mechanistic models of disease. 
        These models enable researchers to simulate and predict the behavior of complex biological systems, providing invaluable insights into disease progression and drug interactions. 
        By leveraging QSP models, we can evaluate potential therapeutic targets, optimize drug design, dosing regimens, and predict patient-specific responses.
        This talk will explore the  role of Quantitative Systems Pharmacology (QSP) models of disease and their impact on the drug development process. </p>

    <p><strong>Biosketch:</strong></p> 
    <p>  Birgit Schoeberl is the Global Head of Modeling and Simulation and Data Science at Novartis BioMedical Research in Cambridge, USA. 
    Her team focuses on leveraging Machine Learning/Artificial Intelligence (ML/AI), Quantitative Systems Pharmacology, and Physiologically Based Pharmacokinetic (PBPK) modeling to accelerate the discovery and clinical development of novel drug candidates. 
    Currently, she is also leading the GenAI initiatives across Biomedical Research. Prior to Novartis Birgit worked for GNS Healthcare and was a founding member of Merrimack Pharmaceuticals. 
    She holds a Ph.D. in Biological Engineering from the Max-Planck Institute in Magdeburg, Germany, and completed a postdoctoral fellowship at MIT. 
    Throughout her career, she has been passionate about integrating diverse computational approaches into the drug development process.</p>
    <p><strong>Video</strong></p>
    <figure class="video-figure">
      <video class="content-video" controls>
      <source src="../../static/videos/my_video.mp4" type="video/mp4">
      Dein Browser unterstützt dieses Video nicht.
      </video>
      <figcaption>My caption text here</figcaption>
    </figure>
  </div>
</div>

<div class="accordion">
  <button class="accordion-button">Feb 12, 2025 - Sophie Fischer-Holzhausen - ESQlabs GmbH</button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Physiologically based kinetic modeling for maternal and female reproductive health </strong><p>

    <p> The women’s health gap refers to the inequities in healthcare between women and men, ranging from healthcare access to the research and limited understanding of sex-specific health needs.
      This presentation aims to demonstrate how Physiologically Based Kinetic (PBK) models can be effectively tailored to female physiology. 
      PBK offers a mechanistic framework for simulating absorption, distribution, metabolism, and excretion (ADME) processes while accounting for female-specific physiological and anatomical characteristics. 
      Through two examples, this presentation will illustrate how this modelling approach can be leveraged to address critical gaps in women’s health research. </p>
    <p> 1.  Pregnancy-PBK (p-PBK) modeling: PBK modeling serves as a valuable tool to predict maternal and fetal exposure to xenobiotic and endobiotic compounds. 
      These predictions have various applications. In drug development, p-PBK modeling can support dose optimization for pregnant populations. 
      Additionally, p-PBK models can aid in assessing the risks of chemical exposure during gestation. 
      We developed a framework for PBK modelling, available in the Open Systems Pharmacology Suite, that provides different complexities of fetal model structure and can account for growth of maternal and fetal organs during long-term exposure/treatments.
    </p>
    <p> 2.  Female reproductive tract (FRT) PBK modeling: Incorporating a detailed representation of the physiology and anatomy of the female reproductive tract can enhance the development of drugs administered via vaginal and uterine routes as well as systemic treatments where the FRT is the target. 
    To support drug development —whether the FRT is central to the drug delivery route or presents the target tissue—we are developing a detailed PBK model of the female reproductive tract, which we are validating with levonorgestrel. Both examples highlight how PBK modeling can address critical gaps in women’s health, from optimizing drug dosing during pregnancy to enhancing treatments targeting the female reproductive tract. 
    Applications extend to rationalizing risk-assessment, studying preeclampsia, microbiome-drug interactions, hormonal fluctuations, and postmenopausal changes, supporting personalized medicine and advancing women’s healthcare.
    </p>

    <p><strong>Biosketch:</strong></p> 
    <p>  Sophie Fischer-Holzhausen, PhD, is a Scientist in Systems Pharmacology at ESQlabs. Her work focuses on advancing computational modeling approaches to address questions in women’s health. </p>
    <p>  In 2019, Sophie graduated with a master’s degree in Biophysics from Humboldt University Berlin. During her studies, she began exploring the intersection between computational science and female physiology, working on a systems biology approach to investigate the regulation of the human menstrual cycle </p>
    <p>  After graduation, Sophie got the opportunity to continue working in this field by joining the Institute of Informatics at the University of Bergen, Norway, as a PhD candidate. 
    Supervised by Prof. Susanna Röblitz, she contributed to the development and extension of mathematical models describing the endocrine regulation of the female reproductive system. 
    In early 2023, Sophie submitted her PhD thesis, "A Matter of Timing – A Modeling-Based Investigation of the Dynamic Behavior of Reproductive Hormones in Girls and Women." - just in time to join AstraZeneca as a Pharmacometrician. 
    Her PhD research included studies demonstrating the potential applications of menstrual cycle models in clinical and pharmaceutical settings, as well as methodological advancements in parameterizing complex models despite limited data availability. </p>
    <p>  In early 2024, she transitioned to ESQlabs, where she now focuses on projects related to female-specific health concerns. 
    Sophie is not only advancing physiologically based kinetic (PBK) modeling strategies but also promotes modeling and simulation to enhance women’s health. </p>
    
  </div>
</div>


<div class="accordion">
  <button class="accordion-button">Jan 15, 2025 - Alexander Skupin - Luxembourg Centre for Systems Biomedicine, University Luxembourg<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Multiscale biology in Parkinson’s disease </strong><p>

    <p> Parkinson’s disease (PD) is the second most prevalent neurodegenerative disease characterized by the loss of dopaminergic neurons in the substantia nigra pars compacta.  Despite intense research our understanding of the underlying disease mechanisms are still incomplete. 
    The talk will showcase how multiscale biology approaches combining longitudinal multiomics experiments of PD patient derived induced pluripotent stem cells with subsequent data integration and mechanistic modelling allows for new insight into disease development. 
    From experimental data, we identify an underlying core network of PD consisting of known and new disease pathways including new candidates for intervention strategies. Two major impairments are mitochondrial dysfunction and dysregulation of calcium signaling. 
    To investigate the crosstalk between these two essential pathways in neurodegeneration, we performed mechanistic multiscale modelling, thermodynamic characterization, and validation experiments to identify an optimal regime for the energetics of cell regulation. </p>

    <p><strong>Video embedden!!!</strong></p>
  </div>
</div>

<div class="accordion">
  <button class="accordion-button">Dec 11, 2024 - Jana Wolf - Max Delbrück Center for Molecular Medicine (MDC) and Freie Universität Berlin (FU) </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Modeling Signal Transduction and Gene Expression in Cancer: From Information Processing to Patient Specific Models </strong><p>

    <p>Signaling pathways regulate the response of cells to external stimuli and can lead to changes in cell fate, e.g. from apoptosis to survival. 
    Important examples are the NF-κB and p53 pathways, both of which are critically involved in the regulation of cell fate decisions after DNA damage and involved in tumor development and treatment. 
    I will here concentrate on models of the NF-kB pathway, discuss the regulation properties underlying complex dynamics and address the elucidation of the cross-talk between the NF-κB and p53 pathway by applying a sub-population-based modeling approach using single cell time-resolved data. 
    I will bridge from the understanding of individual signaling pathways to disease- and patient-specific models with a focus on Diffuse Large B Cell Lymphoma (DLBCL). 
    Important points to address are how to expand and merge pathway models, map disease alterations and derive patient-specific models. </p>

    
  </div>
</div>

##Semester 23|24

<div class="accordion">
  <button class="accordion-button">June 5, 2024 - Thomas Höfer - DKFZ Heidelberg </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Therapy Guided by p53 Dynamics in Single Cells </strong><p>

    <p> Mutations continuously accumulate in somatic tissues and may become selected to drive clonal growth. 
    Typically, we only observe an end result of this evolutionary process – a malignant tumor. 
    In this talk, I will introduce mathematical inference approaches my group has developed to reconstruct the evolutionary dynamics of individual tumors, focusing on cancers of the nervous and hematopoietic systems. 
    Our findings shed light on homeostatic mechanisms that prevent the selection of oncogenic drivers in normal renewing tissues and on the nature of selection pressures encountered by evolving tumors. 
    I will provide an example of how insight into tumor evolution can inform treatment decisions. </p>
  </div>
</div>

<div class="accordion">
  <button class="accordion-button">May 8, 2024 - Galit Lahav - Harvard Medical School, Boston </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Somatic evolution of cancer: From inference to intervention </strong><p>

    <p> Individual human cancer cells often show different responses to the same treatment. 
    In this talk I will share the quantitative experimental approaches my lab has developed for studying the fate and behavior of human cells at the single-cell level. 
    I will focus on the tumor suppressor protein p53, a transcription factor controlling genomic integrity and cell survival. 
    In the last several years my lab has established the dynamics of p53 (changes in its levels over time) as an important mechanism controlling gene expression and cellular outcomes. 
    Specifically, DNA double strand breaks trigger a series of p53 oscillations  allowing DNA repair and recovery, while modulation of p53 dynamics into a sustained response activates genes in terminal programs, including apoptosis and senescence. 
    I will present recent studies from the lab demonstrating how studying the p53 response toradiation and chemotherapy in single cells can guide the design and schedule of combinatorial therapy. 
    I will also present new findings suggesting that p53's post-translational modification state is altered between its first and second pulses of  expression, and the effects these have on gene expression programs over time. </p>
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> Apr 10, 2024 - Andreas Reichel - Head of Preclinical Modeling and Simulation, Bayer AG </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Transforming the discovery and development of targeted protein degraders through application of a comprehensive PK/PD modeling framework </strong><p>

    <p> Targeted protein degradation is an emerging new therapeutic concept that attracts an enormous interest with currently more than twenty compounds in clinical trials. 
    As opposed to conventional target protein inhibitors, it is characterized by a much more complex mode of action with simultaneous kinetics of the drug molecule as well as the target protein, followed by the dynamics of ternary complex formation and target degradation and the subsequent downstream pharmacological responses. 
    The simultaneous occurrence of these processes requires integrative PK/PD modeling to fully comprehend and predict the actions and effects of targeted protein degraders. 
    We therefore have developed a comprehensive mechanistic PK/PD modeling framework that, starting with accurate description of biochemical and cellular degradation in vitro data, allows a priori predictions of target protein degradation profiles in vivo as rational guidance in degrader projects. 
    Our predictions have become a key component of the design of in vivo studies aimed to demonstrate target degradation in animal PD models. 
    They facilitate the optimization and selection of <br>
    i) the most suitable compounds,<br>
    ii) the doses and schedules with the highest likelihood for reaching the desired effect level, and <br>
    iii) the optimal time points for measuring PK and PD. <br>
    The talk will introduce the transformative power of the modeling framework using case examples which illustrate its application for compound progression and the translational understanding of targeted protein degraders. </p>
    
  </div>
</div>



<div class="accordion">
  <button class="accordion-button"> March 6, 2024 - Chengzhe Tian - University of Stuttgart </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Towards a single-cell longitudinal characterization of cancer therapeutic resistance </strong><p>

    <p> Despite the increasing number of effective anti-cancer therapies, successful treatment is limited by the development of drug resistance. 
    While the contribution of genetic factors to drug resistance is undeniable, little is known about how drug-sensitive cells first evade drug action to proliferate in drug. 
    Here we track the responses of thousands of single melanoma cells to BRAF inhibitors and show that a subset of cells escapes drug via non-genetic mechanisms within the first three days of treatment. 
    Cells that escape drug rely on ATF4 stress signaling to cycle periodically in drug, experience DNA replication defects leading to DNA damage, and yet out-proliferate other cells over extended treatment. 
    Together, our work reveals just how rapidly melanoma cells can adapt to drug treatment, generating a mutagenesis-prone subpopulation that expands over time. </p>
    
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> Feb 7, 2024 - Anmar Khadra - McGill University, Montréal </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Characterizing Neural Signaling in Neuropathic Pain Using Computational Modeling </strong><p>

    <p>  Neuropathic pain is a complex condition with a huge unmet medical need. 
    Owing to our incomplete understanding of its perplexing pathology, current therapeutic strategies for treating neuropathic pain remain limited in their efficacy. 
    Computational modeling has emerged as a promising tool to unravel the intricate neural mechanisms underlying neuropathic pain; it has significantly advanced our knowledge of the molecular, cellular, and neural network-level signaling pathways underpinning the disease. 
    The integration of this approach with experimental methods has indeed yielded crucial insights into neuropathic pain pathophysiology, with great potential to inform novel pharmacological and neurostimulation-based treatments for the disease. 
    In this talk, I will provide an overview of our recent computational modeling work that has allowed us to define the role of certain ionic currents in altering neural responses implicated in neuropathic pain. </p>
    
  </div>
</div>

<div class="accordion">
  <button class="accordion-button">Jan 10, 2024 - Mustafa Khammash - ETH Zürich</button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> The Architecture of Biological Adaptation: Deciphering the Balancing Acts Behind Health and Disease </strong><p>

    <p> This presentation delves into the concept of 'robust perfect adaptation' in biological systems, a fundamental mechanism enabling organisms to maintain homeostasis amidst diverse environmental stimuli. 
    We explore how this adaptive resilience is pivotal for health, with its failure often leading to disease states. 
    Focusing on Calcium homeostasis as a case study, the talk will elucidate the essential structural constraints in biological networks that facilitate robust adaptation. 
    These constraints represent universal principles governing biological function and adaptation. 
    The discussion extends to the implications of these findings in systems and synthetic biology, emphasizing the potential to transcend specific system implementations. 
    By understanding the underlying principles of biological adaptation, we can gain comprehensive insights into regulatory complexities in both health and disease. </p>
    
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> Dez 6, 2023 - Laleh Haghverdi - Max Delbrück Center, Berlin<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Inference of temporal dynamics from single-cell high-throughput data </strong><p>

    <p> For several biological processes e.g., development, cell differentiation and disease progression, researchers are interested in understanding the temporal sequence of molecular events, and how they lead to certain final observations on the system. 
    I will discuss a few approaches of dynamical inference from one or a few snapshots of high-throughput single-cell measurements, covering cell state velocities, and response pseudotime inference from time-series data in our recent publications. </p>
    <p><strong> EMBEDD VIDEO!!! </strong></p>
  </div>
</div>

## Semester 22|23


<div class="accordion">
  <button class="accordion-button"> June 7, 2023 - Reiko J. Tanaka - Imperial College London<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> From mechanisms to prediction: Designing personalised treatment strategies for eczema </strong><p>

    <p> Atopic dermatitis (or eczema, AD) is the most common inflammatory skin disease characterised by inflamed, dry and itchy skin leading to substantial quality of life impairment and significant socioeconomic impact. 
    Designing personalised treatment strategies for AD is challenging, given the apparent unpredictability and large variation in AD symptoms and treatment responses within and across individuals. 
    A first step toward developing personalised treatment strategies is to better predict the consequences of possible treatments at an individual level, rather than at population level, to deal with the variability across patients.</p>
    <p> We developed a mechanistic model of AD pathogenesis which provided a coherent mechanistic explanation of the dynamic onset, progression, and prevention of AD, as a result of interactions between skin barrier, immune responses and environmental stressors. 
    Model predictive control using the mechanistic model suggested a possibility for designing personalized treatment strategies. 
    We also adapted the structure of the mechanistic model to real patient data and developed a Bayesian mechanistic model tailored to each individual, that can predict the next day’s AD severity score given their score and treatments used on that day. </p>
    <p> I would like to discuss hopes and challenges in designing personalized treatment strategies, for which detailed dynamic data is required but not often available.</p>
    <figure class="video-figure">
    <iframe
    src="https://nextcloud.mdc-berlin.de/index.php/apps/files/files/4462996?dir=/videos&editing=false&openfile=true"
    width="100%"
    height="360"
    frameborder="0"
    allowfullscreen>
    </iframe>
    <figcaption>Recorded eMed Seminar – June 7, 2023</figcaption>
    </figure>

    
  </div>
</div>

<div class="accordion">
  <button class="accordion-button"> May 10, 2023 - Benjamin Gyori - Director of Machine-assisted Modeling and Analysis, Harvard Medical School </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Scaling up mechanistic modeling of disease processes using automated knowledge assembly </strong><p>

    <p> Computational models of biological mechanisms take substantial human effort to construct and rarely scale to the level of omics datasets, while statistical approaches often do not make use of prior knowledge about mechanisms. 
    At the same time, scientific knowledge is rapidly evolving with about 1.4 million new publications in biomedicine per year. 
    To address these challenges, I present INDRA, an automated knowledge assembly system that integrates multiple text-mining approaches to process scientific literature combined with human-curated databases. 
    INDRA standardizes knowledge extracted from these sources, systematically corrects errors, resolves redundancies, infers missing information, and introduces explicit probability models and machine-learned estimators of confidence to create a coherent knowledge base, up to the scale of all available literature. 
    Detailed simulation models, causal networks, and knowledge graphs are generated from this assembled knowledge to support further analysis. I will present applications of this technology to explaining cancer gene co-dependency data and constructing explanations for experimental observations of drug response in multiple disease areas. 
    This opens up directions toward increasingly automating the scientific discovery cycle with the help of machines building and using models and recognizing surprising observations or proposing new hypotheses. </p>
    
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> April 5, 2023 -  Julio Saez-Rodriguez - University of Heidelberg<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Dynamic logic models complement machine learning for personalized medicine </strong><p>

    <p>  Multi-omics technologies, and in particular those with single-cell and spatial resolution, provide unique opportunities to study the deregulation of intra- and inter-cellular signaling processes in disease. 
    I will present recent methods and applications from our group toward this aim, focusing on computational approaches that combine data with biological knowledge within statistical and machine learning methods. 
    This combination allows us to increase both the statistical power of our analyses and the mechanistic interpretability of the results. These approaches allow us to identify key processes, that can be in turn studied in detailed with dynamic mechanistic models. 
    I will then present how cell-specific logic models, trained with measurements upon perturbations, can provides new biomarkers and treatment opportunities. 
    Finally, I will show how, using novel microfluidics-based technologies, this approach can also be applied directly to biopsies, allowing to build mechanistic models for individual cancer patients, and use these models to prose new therapies.</p>
    <figure class="video-figure">
      <video class="content-video" controls>
      <source src="../static/videos/eMed_Saez Reodriguez_23 04 05.mp4" type="video/mp4">
      Dein Browser unterstützt dieses Video nicht.
      </video>
      <figcaption></figcaption>
    </figure>
    
  </div>
</div>



<div class="accordion">
  <button class="accordion-button"> February 1, 2023 - Marcus Rosenblatt & Jens Timmer - Institute of Physics, University of Freiburg<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Mechanistic insights into sensitization and desensitization of the Interferon α signal transduction pathway </strong><p>

    <p>   As a key component of the innate immune system, Interferon alpha (IFNα) orchestrates the antiviral response in hepatocytes. 
    The IFNα signal transduction pathway is known to desensitize upon activation which constitutes a major problem for the usage of IFNα as treatment against chronic viral infections or as an anti-tumor drug. 
    However, the mechanisms that lead to this desensitization remain poorly understood.<br>
    Here, an ODE model is presented that describes the biochemical reaction network of IFNα signaling in different hepatoma cell lines as well as primary human hepatocytes (PHH). 
    The calibrated model shows that besides a dose-dependent desensitization mediated by the negative feedback components SOCS1 and USP18 that act at the receptor level, the signaling pathway can also show (hyper-)sensitization in consequence of an upregulation of the intra-cellular components IRF9 and STAT2.<br>
    The model predicted the dose-dependent dynamics of transcriptionally active complexes in the signaling pathway and their effect on mRNA production, as shown by independent validation experiments. Furthermore, the model-based analysis of measurement data from PHH unraveled that each cell system establishes a particular dose-depending sensitization behavior whose shape is strongly determined by the abundance of the feedback components USP18 and STAT2. 
    We show that our findings help to understand the dynamics of production of Interferon Stimulated Genes (ISGs) which exert numerous antiviral effector functions, and serve as a basis for a patient-individual optimization of the antiviral response upon IFNα stimulation.<br>
    </p>
    <p><strong> VIDEO embedden! </strong></p>
  </div>
</div>

 

<div class="accordion">
  <button class="accordion-button"> January 11, 2023 - Jae Kyoung Kim - Dept. of Mathematical Sciences KAIST, South Korea<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Modeling and Inference for Biological Systems with Hidden Components </strong><p>

    <p>   Despite dramatic advances in experimental techniques, many facets of intracellular dynamics remain hidden or can be measured only indirectly. 
    In this talk, I will describe strategies to develop mathematical models with hidden parts: replacement of hidden components with either time delay or quasi-steady-state. 
    Then, I will illustrate how the simplification with the time delay can be used to understand the processes of protein synthesis, which involves multiple steps such as transcription, translation, folding, and maturation, but typically whose intermediates proteins cannot be measured. 
    Furthermore, I will illustrate how the simplification with the quasi-steady-state can be used to develop an accurate method to estimate drug clearance, which occurs in multiple steps of metabolism, which greatly improved the canonical approach used in more than 65,000 published papers for last 30 years. 
    Finally, I will describe an inference method, GOBI (General Model-based Inference), that identifies hidden regulatory biochemical connections from time-series data. 
    This method adopts the advantage of model-free inference methods (broad applicability) and model-based inference methods (accuracy). 
    </p>
    <p><strong> VIDEO embedden! </strong></p>
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> December 7, 2022 -  Ina Koch - University Frankfurt <strong>including slides</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Semi-quantitative modeling in systems biology: the Petri net formalism </strong><p>

    <p> Modeling disease pathways requires the integration of various data of different type and quality. Whereas the data needed for deterministic approaches, such as ODE-based modeling, are often rare or not available, a huge amount of qualitative data have been generated by high-throughput methods in the last years. This causes the need for developing and applying new modeling approaches. Besides Boolean modeling, semi-quantitative modeling using Petri nets has been successfully applied to model biochemical systems, covering metabolic systems, signal transduction pathways, gene-regulatory pathways and hybrid systems.<br>
        In the talk, we will first introduce the basics of the Petri net formalism and its static and dynamic analysis techniques. 
        The techniques range from basic graph-theoretical considerations and integer linear programming techniques to dynamic simulations. 
        We will focus on invariant-based analysis approaches, in particular on transition invariants and Manatee invariants. 
        We will illustrate the introduced terms using a case study of signal transduction in the NF-kB pathway before considering an exhaustive Petri net model of the TNFR1-induced signaling pathway, which covers cell survival, apoptosis and necroptosis processes. 
        We will present the main results and additionally discuss the results of in silico knockout analysis based on the Petri net model. 
    </p>
    <p><strong> SLIDES embedden! </strong></p>
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> November 9, 2022 - Ralf Steuer - Institute for Theoretical Biology, Humboldt University Berlin </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Models of metabolism for biomedical research </strong><p>

    <p> Cellular metabolism is the ultimate driving force of life. Metabolism provides the chemical energy required to run cellular processes, ensures cellular functions and removal of byproducts, and synthesises the building blocks for growing cells. 
        Metabolic processes and the resulting metabolite levels can be interpreted as "end products" of cellular regulatory processes, and metabolic dysfunctions are the underlying cause of a multitude of diseases, including cancer. 
        The talk aims to discuss the current dichotomy between small-scale kinetic models, and comprehensive genome-scale metabolic reconstructions of metabolism in the context of biochemical research. In particular, while genome-scale reconstructions offer unprecedented possibilities to study the organization of human metabolism, their analysis is typically restricted to stoichiometric properties and stationary fluxes. In contrast, kinetic models of metabolism based on ordinary differential equations allow for a dynamic description, but their reconstruction requires extensive (and currently often not available) knowledge about enzyme-kinetic parameters. The talk will explore intermediate methods that<br>
        (a) seek to derive smaller coarse-grained models from genome-scale reconstructions, and <br> 
        (b) use Monte-Carlo methods to account for unknown or uncertain parameters. 
    </p>
  </div>
</div>


##Semester 22



<div class="accordion">
  <button class="accordion-button"> June 15, 2022 - Bernhard Steiert - Roche Pharmaceutical Research and Early Development, Roche Innovation Center Basel, Basel, Switzerland</button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> How mathematical models support clinical drug development: an industry perspective </strong><p>

    <p> Mathematical modelling is a key component of drug development and is applied at all stages of a molecule progressing along the value chain. 
        In clinical development, it is essential to find the right dose for every patient, which is composed of the right amount of drug and the right treatment interval for the right patient population. 
        Modelling is also used to leverage and extrapolate the available data, informing decisions on which programs to accelerate and which ones to stop.<br>
    
        In this talk, an overview of the Roche Pharma Research and Early Development (pRED) setup for modelling will be provided. 
        Typical models used in the pharmaceutical industry will be exemplified and, based on those, more general modelling themes are outlined and discussed. 
        Involvement of key internal and external stakeholders is crucial for impacting decisions with modelling. 
        I will summarize who they are and which communication strategies are most suitable for building trust in modelers and their models. 
        The talk will close with examples where modelling was impactful and helped to bring safe and efficacious medicines to patients faster.
    </p>
  </div>
</div>



<div class="accordion">
  <button class="accordion-button"> May 4, 2022 - Gavin Fullstone (Stuttgart Research Centre Systems Biology) & Markus Morrison (Institute of Cell Biology and Immunology) - University of Stuttgart<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Easy implementation and acceleration of spatial agent-based modelling of biological processes using rule-based modelling </strong><p>

    <p> Agent-based modelling is particularly adept at modelling complex features of cell signalling pathways, where heterogeneity, stochastic and spatial effects are important, thus increasing our understanding of decision processes in biology in such scenarios. 
    However, agent-based modelling often is computationally prohibitive to implement. Parallel computing, either on central processing units (CPUs) or graphical processing units (GPUs), can provide a means to improve computational feasibility of agent- based applications but generally requires specialist coding knowledge and extensive optimisation. 
    In this seminar, we will demonstrate how we implemented and utilise a rule-based modelling approach to define particle-based models that can be flexibly parallelised on either CPU or GPU-architecture. 
    In comparison with an established particle-based simulator, we achieve speed ups of >650-fold on modern GPUs whilst maintaining robust simulation. Additionally, we will demonstrate how we apply stochastic hybrid approaches to model the activity of multi-protein complexes and their contribution to cell signalling outcomes. 
    In introducing these approaches, we will simultaneously present specific examples where we have successfully applied these approaches to help us understand fundamental biological processes, predict outcomes in biologically relevant scenarios and in the design of targeted therapeutics.
    </p>
    <p><strong> VIDEOS embedden! </strong></p>
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> April 6, 2022 - Martin Falcke (Max-Delbrück-Centrum für Molekulare Medizin - MDC, Berlin) & Lutz Brusch (Research Group Leader,Center for Information Services and High Performance Computing -ZIH-, Technische Universität Dresden)<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Multi-scale modelling </strong><p>

    <p> Biological systems comprise many structural levels represented by molecules, cell organelles, cells, tissues, organs, organisms, populations and ecological levels. 
    Each structural levels has its typical time and length scale from fs and nm on molecular level to km and years on ecological level. 
    Multiscale modelling refers to mathematical models describing dynamics on different structural levels. Here, several structural levels and several scales may be accounted for either by a single equation or by multiple coupled equations each comprising only one scale. 
    We present examples of both approaches.<br> 
    In the first case, we present reaction diffusion systems for cardiac myocytes taking behaviour of individual ion channels into account and simulating whole cells at the same time. 
    The mathematical problems and methods addressing them will be explained. For the second case, we introduce coupling of processes between intracellular, cellular and tissue scales. 
    Such coupling requires that parameters of model processes on one scale get driven by the dynamical variables of model components at another scale. 
    We demonstrate such model integration across multiple scales in the software framework Morpheus <a href="https://morpheus.gitlab.io">Morpheus</a> and discuss the emergence of complex 'morphodynamic' behaviour that cannot be addressed by simpler models at a single scale.
    </p>
    <p><strong> VIDEOS embedden! </strong></p>
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> March 2, 2022 - Kevin Thurley - Biomathematik, Institut für Experimentelle Onkologie, Universitätsklinikum Bonn<br> <strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Response-Time Modelling of Cell-Cell Interaction Dynamics </strong><p>

    <p> Cell-to-cell communication networks have critical roles in coordinating diverse organismal processes, such as tissue development or immune cell response. 
    However, compared with intracellular signal transduction networks, the function and engineering principles of cell-to-cell communication networks are far less understood. 
    Major complications include: cells are themselves regulated by complex intracellular signaling networks; individual cells are heterogeneous; and output of any one cell can recursively become an additional input signal to other cells. 
    In this talk, I will introduce a framework that treats intracellular signal transduction networks as "black boxes" with characterized input-to-output response relationships (Thurley K, Wu LF, Altschuler SJ, Cell Systems 2018, <a href="https://pubmed.ncbi.nlm.nih.gov/29525203/">DOI: 10.1016/j.cels.2018.01.016</a>). 
    The approach can be used to predict communication network structure using experimentally accessible input-to-output measurements and without detailed knowledge of intermediate steps. 
    Here, response-time modeling will be presented in the context of data-driven analysis of T cell differentiation and expansion in chronic viral infections.</p>
    <p><strong> contact: </strong></p>
    <p> <a href="https://www.thurleylab.org">ThurleyLab</a>
    </p>
    <p><strong> VIDEOS embedden! </strong></p>
  </div>
</div>


<div class="accordion">
  <button class="accordion-button"> February 2, 2022 - Prof. Dr. Jan Hasenauer - LIMES Institute, Universität Bonn <br><strong>including video</strong> </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong> Large-scale pathway models for biomedical research </strong><p>

    <p> Pathway models are powerful tools in modern life sciences. 
    They are based on knowledge of individual biological pathways and can be integrated into large-scale models to capture crosstalk. 
    In this seminar, I will briefly introduce the modeling approach and methods used to create large-scale models. 
    I will then present a large-scale model of cancer signal transduction with thousands of biochemical species and reactions as an example application. 
    Variants of this model have been used for multi-omics analyses in murine model systems and human cell lines. 
    We have developed workflows for parameterization and analysis of these models.
    I hope to discuss these approaches and future directions. 
    </p>
    <p><strong> VIDEOS embedden! </strong></p>
  </div>
</div>



<div class="accordion">
  <button class="accordion-button"> January 12, 2022 - Prof. Dr. Olaf Wolkenhauer (Dept of Systems Biology & Bioinformatics), Shailendra Gupta (Systems Biology & Bioinformatics) & Matti Hoch (Systems Biology & Bioinformatics) - University of Rostock </button>
  <div class="accordion-content">
    <p><strong>Location:</strong> Zoom</p>
    <p><strong>  Using Disease Maps in Biomedical Research </strong><p>

    <p> Disease maps are interactive, web-based representations of the phenotypic, cellular, and molecular processes underlying complex diseases in the form of knowledgebase. 
    Disease maps provide a platform to integrate heterogeneous clinical and experimental data for formulating hypotheses on diagnostic, prognostic, and/or therapeutic markers using systems biology and bioinformatics-based methods. 
    In the seminar, we shall contrast the disease map approach with small-scale pathway modelling efforts and present examples from cancer systems biology and demoing the Atlas of Inflammation Resolution (AIR) <a href="https://air.bio.informatik.uni-rostock.de">AIR</a> .
    The AIR contains 40 manually curated submaps including > 6T elements each of which are manually annotated. In addition, the AIR is enriched with regulatory layers of microRNAs, lncRNAs, and transcription factors that form a large molecular interaction map (MIM) with >23,000 molecular interactions. 
    We developed workflows that facilitate the integration and analyses of multi-omics data directly on the disease maps and enable users to perform in silico perturbation experiments for formulating data-driven hypotheses. 
    In the seminar, we hope for a discussion in which we compare the “disease map approach” with other pathway and network modelling approaches.
    </p>
  </div>
</div>




